gridpath/transmission/operations/operational_types/tx_simple.py

```diff
@@ -327,6 +373,26 @@ def max_losses_to_rule(mod, l, tmp):
             * mod.Tx_Availability_Derate[l, tmp] \
             * mod.tx_simple_loss_factor[l]
 
+
+def min_transmit_power_rule(mod, l, bt, h):
+    """
+    **Constraint Name**: TxSimple_Min_Transmit_Power_Constraint
+    **Enforced Over**: TX_SIMPLE_OPR_TMPS_W_MIN_CONSTRAINT
+
+    Transmitted power should exceed the defined minimum transmission power in
+    each operational timepoint.
```

Comment:
@Janie115, could you add some explanation of the logic in the docstring here. I'm still not sure what we are trying to do. Is `tx_simple_min_transmit_power_mw` the amount that must be transmitted in _either_ direction?

Reply:
Yes, it is the amount that must be transmitted in one direction or the other. If the min transmit power is positive, then the transmitted power must exceed this value in the positive direction. If the min transmit power is negative, then the transmitted power must exceed this value in the negative direction.

Reply:
Got it, sorry for my confusion. I have a few thoughts I'll share in a bit.

Reply:
I was confused earlier by the use of "min" in the parameter name and am realizing about defaulting to negative infinity probably didn't make much sense. I suggest the following: let's have two parameters here, `tx_simple_min_flow_mw` that defaults to `Negative_Infinity` and `tx_simple_max_flow_mw` that defaults to `Infinity`. We'll then have two constraints `TxSimple_Transmit_Power_MW>= tx_simple_min_flow_mw` and `TxSimple_Transmit_Power_MW<= tx_simple_max_flow_mw`. I think this is a more intuitive way to achieve the same effect without the `if` logic and it allows for extra flexibility with bounding on both sides and even forcing a particular level of flow. Thoughts?